index.html

```diff
@@ -12,10 +12,101 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    
+    <nav class ="navbar">
+        <div class="content" >
+            
+            <ul class="link">
+                <li class="active">Meet Karma</li>
+                <li><a href="How it works">How it works</a></li>   
+                <li><a href="Store">Store</a></li>   
+                <li><a href="Blog">Blog</a></li>   
+                <li><a href="Help">Help</a></li>
+                <li><a href="Login">Login</a></li>        
+          
+            </ul>
+              <img class="header-image" src="/img/karma-logo.svg" alt="image logo" width="26" /> 
+
+         
+        </div>  
+            
+            
+    </nav>
+
+    
+    <section class="introduction">
+        <h1>Introducing Karma</h1>
+        <h3>bring wifi with you, Everywhere you go.</h3>
+        <div class="btn">
+            <button>Learn more</button>
+        </div> 
+        
+    </section>
+    
+    
+    <section class="cases">
+        <div class="content">
+            <h2>Everyone needs a little a Karma.</h2>
+            
+            <ul class="devices">
+              <li>
+                 <img width="127" height="127" src="img/icon-devices.svg" alt="internet for all device"/>
+                    <div class="caption">
+                     <h3>internet for all devices</h3>
+                    </div>
+                
+                </li>
+
+                <li>
+                  <img width="127" height="127" src="img/icon-coffee.svg" alt="Boost your productivity"/>
+                    <div class="caption">
+                     <h3>Boost your productivity</h3>
+                    </div>
+  
+                </li>
+
+                <l1>
+                   <img width="127" height="127" src="img/icon-refill.svg" alt="Pay as you go"/>
```

Comment:
I would avoid using inline attributes for image dimensions like this and instead handle this in css.

i.e

li img {
  width:127px;
  height:127px;
}

Reply:
Its best to separate styling from html as much as possible as it will mean its easier to make changes to styling in the future. You should be able to swap stylesheets easily to give the code a 'reskin' without having to edit the html ideally.